keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ArtifactRepository.kt

```diff
@@ -208,6 +220,46 @@ interface ArtifactRepository : PeriodicallyCheckedRepository<DeliveryArtifact> {
    * different values.
    */
   override fun itemsDueForCheck(minTimeSinceLastCheck: Duration, limit: Int): Collection<DeliveryArtifact>
+
+  /**
+   * Returns true if the new version is lower than the existing version.
+   * Note: the artifact comparitors are decending by default
+   */
+  fun isLower(artifact: DeliveryArtifact, new: String, existing: String): Boolean =
```

Comment:
suggestion - maybe use `old` and `new` instead of `low` and `high`? I saw that you used these terms in other places in the code and it made sense to me better..

Reply:
You're right, that's sooooo much better.